### PR TITLE
Add dry-run argument.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This will only run a dry-run import instead of committing the import.
However since the aptly output is not displayed the utility of this
option is limited.

### CODE REVIEW

File: scripts/aptly/aptly_importer.py [hunk 1]
```diff
@@ -319,18 +324,19 @@ def run(self):
 
 def main():
     """
-    Usage: python3 aptly_importer.py <config_file>
+    Usage: python3 aptly_importer.py [--dry-run] <config_file>
```
Comment (j-rivero) [inline scripts/aptly/aptly_importer.py]: Before starting to look code changes I was expecting that `--dry-run` means doing nothing on the system but is only adding `-dry-run` to the import from mirror to repo. Can we rename the argument to make it clearer that dry-run will only be applied to import from mirror to repo?

**Comment by j-rivero** [comment]
Another point would be: when running with `--dry-run`, do we want to stop the steps of creating snapshots and publish them?

File: scripts/aptly/aptly_importer.py (hunk 1 above)
Comment (j-rivero) [reply scripts/aptly/aptly_importer.py]: I committed a change for the name in be74c84

**Comment by j-rivero** [comment]
> Another point would be: when running with `--dry-run`, do we want to stop the steps of creating snapshots and publish them?

I committed the code to stop processing the actions if simulation is enabled in be74c84. Feel free to revert if you have a better alternative or think it is wrong.

File: scripts/aptly/aptly_importer.py (hunk 1 above)
Comment (nuclearsandwich) [reply scripts/aptly/aptly_importer.py]: Good call.

**Pull request merged**